scripts/common.py

```diff
@@ -1,7 +1,8 @@
 import os
 from msbase.subprocess_ import try_call_std
 from msbase.utils import getenv
-
+os.environ["MARKII_DIR"] = os.getcwd() + os.sep + "markii"
```

Comment:
You should not change environ within the program. These should be instructions for users or scripts.